### PR TITLE
PE-67 | Implemented a Side Menu Service

### PR DESCRIPTION
**Pull request by tonynguyen98**
**What I Did:*
- Used the UserById Service created by @davidsquines to implement dynamic data into the side menu.
- Removed getAllUsers Service, since that is unneeded.

**How to Test:**
1. Sign in using any account.
2. Open the Side Menu and the account's name and email should show up.
3. Check all pages of the Side Menu.

### CODE REVIEW

**Review by dominikyz — APPROVED** [review]
At first i couldn't see any information when signed in with my google. Deleted account from firebase. Logged in again with google, provided name etc. Aaaand everything works now. Regular email works good too. Great job!

File: lib/components/side_menu.dart [hunk 1]
```diff
@@ -1,50 +1,93 @@
...
+}
+
+class _SideMenuState extends State<SideMenu> {
+  UserModel user = new UserModel(
+    firstName: '',
+    lastName: '',
+    email: '',
+    pictureUrl: '',
+  );
+
```
Comment (txt020) [inline lib/components/side_menu.dart:28]: Where do I go to change my profile picture? Can you add a button where I can add a profile picture?

**Review by txt020 — CHANGES_REQUESTED** [review]
Based off what you did everything works as intended great job!!

@tonynguyen98 Small request can you add a feature where the use can change their profile picture?

File: lib/components/side_menu.dart (hunk 1 above)
Comment (tonynguyen98) [reply lib/components/side_menu.dart:28]: That's an unrelated task to the side menu and it's currently in Open in our JIRA board.

**Review by txt020 — APPROVED** [review]
Looks good!

**Review by ayeejane — APPROVED** [review]
Works as intended.

**Pull request merged**